src/test/java/com/trycloud/stepDefinitions/DeletedFile_stepDef_TD.java

```diff
@@ -3,61 +3,128 @@
 import com.trycloud.pages.*;
 import com.trycloud.utility.*;
 import io.cucumber.java.en.*;
+import io.cucumber.java.en_old.*;
+import org.apache.commons.logging.*;
+import org.apache.logging.log4j.*;
 import org.junit.*;
+import org.openqa.selenium.*;
+import org.openqa.selenium.interactions.*;
+
+import java.util.*;
+import java.util.NoSuchElementException;
+
+import static org.junit.Assert.*;
+
 
 public class DeletedFile_stepDef_TD {
+    Actions action=new Actions(Driver.getDriver());
     DeletedFilesPage deleted=new DeletedFilesPage();
+    Logger LOG = LogManager.getLogger();
     @When("user clicks on Deleted files under All files")
     public void user_clicks_on_deleted_files_under_all_files() {
+        //special step to create file delete talk and Dilo file
+        //delete Talk folder
+        deleted.fileEllipses("Talk").click();
+        deleted.deleteFolder.click();
```

Comment:
after deleting the folder, page should be refreshed to create a new folder

there might be a potential defect

![Screenshot 2024-11-12 at 12 00 11 AM](https://github.com/user-attachments/assets/d8ded333-3bd8-46e7-911d-c33838df01c3)